x/twap/logic_test.go

```diff
@@ -74,10 +90,10 @@ func (s *TestSuite) TestGetSpotPrices() {
 
 	for name, tc := range testCases {
 		s.Run(name, func() {
+			ctx := s.Ctx.WithBlockTime(tc.blockTime)
```

Comment:
what's the significance of setting ctx with blocktime when there are no arguments to compare it with? (what / why are we setting blocktime for?)

Reply:
For case getting sp err & no previous record. Actually, this case never happen cause pool already have a record when created. So I use blockTime before pool creation's time to make sure no record.